### PR TITLE
Add Sort-By Action to Search Activity

### PR DESCRIPTION
**Pull request by alonalbert**
Support a Sort-By action in Search Activity toolbar.

### CODE REVIEW

File: app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java [hunk 1]
```diff
@@ -85,6 +85,8 @@
 	private List<SearchSetting> searchSites;
 	private SearchSetting lastUsedSite;
 	private String lastUsedQuery;
+	private MenuItem sortByAdded;
```
Comment (erickok) [inline app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java]: We shouldn't store menu items this way. Instead, just invalidate the menu after changing the sort order, such that onCreateOptionsMenu is called again.

File: app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java [hunk 2]
```diff
@@ -314,4 +324,23 @@ protected void downloadSearchModule() {
 		startActivity(new Intent(Intent.ACTION_VIEW, Uri.parse("http://www.transdroid.org/latest-search")));
 	}
 
+	@OptionsItem(R.id.action_sort_added)
+	protected void sortByDateAdded() {
+		if (sortByAdded.isChecked()) {
```
Comment (erickok) [inline app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java]: For this logic you could just check `applicationSettings.getLastUsedSearchSortOrder()`.

File: app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java (hunk 1 above)
Comment (alonalbert) [reply app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java]: done

File: app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java (hunk 2 above)
Comment (alonalbert) [reply app/src/main/java/org/transdroid/core/gui/search/SearchActivity.java]: done

**Pull request merged**